_posts/2022-01-17-cockpit-261.md

```diff
@@ -0,0 +1,95 @@
+
+Similarly there are two redirect target types: a local channel and a peer. These are handled by different implementations of the `CockpitRedirect` interface - `CockpitChannelRedirect` for local channels (holding a reference to the corresponding `CockpitChannel` object) and `CockpitPeerRedirect` for peers (holding a reference to the transport and the channel name).
+
+Redirection of a channel is attempted on all bridges - the shortest path then takes precedence (e.g. if there is a redirect between two channels on the same peer, it happens on both the parent bridge and the peer, but effectively the data is redirected on the peer and nothing arrives on the parent). This also means it is compatible with older versions of cockpit-bridge - they simply ignore the redirect option and the redirect is done on the parent bridge (this can obviously be slower, but it will work).
+
+I hope this makes sense - it seems like a small, but useful feature to me. (For potential uses see the linked issue.)
+
+
+## Podman: Add pause/resume to containers
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/870>
+{:.note}
+
+Bring containers to the similiar feature parity to podman pods by adding
+pause/resume options. Also make the order of the menu options the same
+for pods and containers and add "Force restart" to the pod actions.
+
+## Always pull the latest image when creating a new container
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit-podman/pull/859>
+{:.warn}
+
+When a local image `fedora:latest` is available but a newer version is available on the image registry, checking the "pull latest image" checkbox will pull the latest version before creating the new container
+
+![screenshot of ## always pull the latest image when creating a new container]({{ site.baseurl }}/images/261-always-pull-the-latest-image-when-creating-a-new-container.png)
+
+
+## Try it out
+
+Cockpit 261 and cockpit-podman 40 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/261)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/40)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-podman)
+* [cockpit-podman Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-podman)
+
+*[CPU]: Central Processing Unit, the "brain" of a computer
```

Comment:
also: lol.  seriously?  I don't think our target audience needs this if we're freely discussing pods, containers, image tags, unmounting busy filesystems, etc. all without further explanation.